doc/user_guide/model.rst

```diff
@@ -241,6 +237,24 @@ the following template to suit your needs:
             """
             return x
 
+Define components from a fixed-pattern
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :py:class:`~._components.scalable_fixed_pattern.ScalableFixedPattern`
+component enables fitting a pattern (in the form of a
+:py:class:`~._signals.signal1d.Signal1D` instance) to data by shifting 
+(:py:attr:`~._components.scalable_fixed_pattern.ScalableFixedPattern.xshift`)
```

Comment:
The link to this attribute doesn't work?

Reply:
Actually there was a typo there that I've just fixed. However, the links can't work because the attributes are created by the constructor.